tao/tao_bridge/tf_compatible.h

```diff
@@ -11,6 +11,7 @@
 
 #pragma once
 
+#include "tensorflow/core/framework/resource_mgr.h"
```

Comment:
Just curious why to add this include.

Reply:
we encounter a failure on arm without this header. I do not know why it's ok on x86. Maybe compiler version is different?